stable/rabbitmq/templates/configuration.yaml

```diff
@@ -16,7 +16,15 @@ data:
     default_pass=CHANGEME
 {{ .Values.rabbitmq.configuration | indent 4 }}
 {{ .Values.rabbitmq.extraConfiguration | indent 4 }}
+{{- if .Values.rabbitmq.tls.enabled }}
+    ssl_options.verify={{ .Values.rabbitmq.tls.sslOptionsVerify }}
+    listeners.ssl.default={{ .Values.service.SSLPort }}
+    ssl_options.fail_if_no_peer_cert={{ .Values.rabbitmq.tls.failIfNoPeerCert }}
+    ssl_options.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    ssl_options.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    ssl_options.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
+{{ end }}
 {{ if .Values.rabbitmq.advancedConfiguration}}
   advanced.config: |-
 {{ .Values.rabbitmq.advancedConfiguration | indent 4 }}
-{{ end }}   
+{{ end }}
```

Comment:
Small improvement you can apply in every `{{ end }}` you added:
```diff
- {{ end }}
+ {{- end }}
```

Reply:
replaced